clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/common/AzureUtilities.groovy

```diff
@@ -261,5 +235,5 @@ class AzureUtilities {
     url = url.replace(" ", "%20")
 
     url
-  }
+  }*/
```

Comment:
We should remove this (and the test) since we don't need it. That is unless we are still expecting to make the REST call ourselves ;-)